compare.py

```diff
@@ -0,0 +1,79 @@
+from preprocess import request_pkg_info
+from pprint import PrettyPrinter
+
+# TODO
+# Handle the case that the conditions are NOT overlap
+# Handle the case `==`
+# Remove this todo
+def get_possile_pkg_by(pkg_name, conditions):
+    """Get possible packages by a list of conditions.
+
+    Parameters:
+        pkg_name : str
+        conditions : list
+
+    Returns:
+        list
+
+    Warning:
+        This function is right for the case that the conditions overlap.
+    """
+    releases = list(request_pkg_info(pkg_name)['releases'].keys())
+    for condi in conditions:
+        if condi[1] not in releases:
+            continue
+        else:
+            # if condi[0] == '==':
+            #     possible_vers.append(condi[1])
+
```

Comment:
Why do you comment this?

Reply:
Honestly, I don't remember why I commented that. It could be something that I forgot to discuss and to add to a to-do list.

Reply:
Hmmm. If there is one condition in the list which is `A==6.9`, it will be always only one conditions. If I uncomment it, it should still work with for that case. I think I commented it maybe because I wanted to handle the case in different way.